common/environment/environment.cpp

```diff
@@ -1036,14 +1149,34 @@ void CLocalEnvironment::buildDropZoneCache() const
                 Owned<IConstEnvBase> cached = new CConstDropZoneInfo((CLocalEnvironment *) this, &it->query());
                 cache.setValue(x.str(), cached);
             }
-            name = it->query().queryProp("@computer");
-            if (name)
+            const char * computer = it->query().queryProp("@computer");
+            if (computer)
```

Comment:
It comes from an XML file, which can be altered by a simple text editor. So I think it can be undefined. (This check was there before I change the 'name' to 'computer' to avoid confusing by use same variable for two different thing)